src/api/entities/Portfolio/index.ts

```diff
@@ -130,4 +135,30 @@ export class Portfolio extends Entity<UniqueIdentifiers> {
 
     return values(assetBalances);
   }
+
+  /**
+   * Retrieve the custodian of this portfolio
+   */
+  public async getCustodian(): Promise<Identity> {
```

Comment:
Should return null if there is no custodian. We will implement `getAuthorizer` in the future, which will do what you were trying to do here